modules/cnf-getting-the-dpll-firmware-version-for-intel-800-series-nics.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * networking/ptp/configuring-ptp.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cnf-getting-the-dpll-firmware-version-for-intel-800-series-nics_{context}"]
+= Getting the DPLL firmware version for the CGU in an Intel 800 series NIC
+
+Get the digital phase-locked loop (DPLL) firmware version for the Clock Generation Unit (CGU) in an Intel 800 series NIC by opening a debug shell to the cluster node and querying the NIC hardware.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in as a user with `cluster-admin` privileges.
+
+* You have installed an Intel 800 series NIC in the cluster host.
+
+*  You have installed the PTP Operator on a bare-metal cluster with hosts that support PTP.
+
+.Procedure
+
+. Start a debug pod by running the following command:
++
+[source,terminal]
+----
+$ oc debug node/<node_name>
+----
++
+where:
++
+--
+<node_name>:: Is the node where you have installed the Intel 800 series NIC.
+--
+
+. Check the CGU firmware version in the NIC by using the `devlink` tool and the bus and device name where the NIC is installed.
+For example, run the following command:
++
+[source,terminal]
+----
+sh-4.4# devlink dev info <bus_name>/<device_name> | grep cgu
+----
++
+where:
++
+--
+<bus_name> :: Is the bus where the NIC is installed. For example, `pci`.
+<device_name> :: Is the NIC device name. For example, `0000:51:00.0`.
+--
++
+.Example output
+----
+cgu.id 36 <1>
+fw.cgu 8032.16973825.6021 <2>
+----
+<1> CGU hardware revision number
+<2> The DPLL firmware version running in the CGU, where the DPLL firmware version is `6201`, and the DPLL model is `8032`.
+The string `16973825` is a shorthand representation of the binary version of the DPLL firmware version (`1.3.0.1`).
++
+[NOTE]
```

Comment:
I think this is attaching to the step rather than the firmware version callout, but i messed around with open blocks a bit and couldn't figure it out. I don't think it's hurting clarity at all.